### PR TITLE
Test broken links

### PR DESCRIPTION
**Pull request by barbaraperic**
## Note
- added cypress tests in order check for broken links
- let me know what you think, happy to explore the Github action suggestion as well. Just thought this might be an easier solution.

## How to test
- run `npx cypress open` and check the logs

## Visuals
Screenshot 2024-07-09 at 15 18 10

### CODE REVIEW

**Review by mirhamasala — COMMENTED** [review]
1. Why is it easier?
2. Does this mean we'd be checking 160+ blog posts, 80+ events, and 180+ ecosystem project pages on each PR and build?
3. Also, what if there are multiple broken links on a page? It seems the tests stop when a broken link is found (per page test)?

**Comment by barbaraperic** [comment]
> @barbaraperic
> 
> 1. Why is it easier?
> 2. Does this mean we'd be checking 160+ blog posts, 80+ events, and 180+ ecosystem project pages on each PR and build?
> 3. Also, what if there are multiple broken links on a page? It seems the tests stop when a broken link is found (per page test)?

Let me try something, I'll get back to you with the answers :)

**Review by mirhamasala — APPROVED** [review]
Super excited about this! ❤️ What do you think about adding skip commands for now so that we can get this PR and the template PR merged, while you work on fixing the broken links?

File: cypress/e2e/about_page_spec.cy.ts [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('About Page', function () {
-  testPageMetadata({ path: PATHS.ABOUT, useAbsoluteTitle: true})
+describe('About Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.ABOUT,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/about_page_spec.cy.ts:9]: It's the default.

```suggestion
```

File: cypress/e2e/blog_page_spec.cy.ts [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('Blog Page', function () {
-  testPageMetadata({ path: PATHS.BLOG, includesFeaturedEntry: true, useAbsoluteTitle: true})
+describe('Blog Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.BLOG,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/blog_page_spec.cy.ts:9]:
```suggestion
```

File: cypress/e2e/ecosystem_explorer_page_spec.cy.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { PATHS } from '../../src/app/_constants/paths'
+import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
+
+describe('Ecosystem Explorer Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.ECOSYSTEM_EXPLORER,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/ecosystem_explorer_page_spec.cy.ts:9]:
```suggestion
```

File: cypress/e2e/events_page_spec.cy.ts [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('Events Page', function () {
-  testPageMetadata({ path: PATHS.EVENTS, includesFeaturedEntry: true, useAbsoluteTitle: true})
+describe('Events Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.EVENTS,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/events_page_spec.cy.ts:9]:
```suggestion
```

File: cypress/e2e/governance_page_spec.cy.ts [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('Governance Page', function () {
-  testPageMetadata({ path: PATHS.GOVERNANCE, useAbsoluteTitle: true})
+describe('Governance Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.GOVERNANCE,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/governance_page_spec.cy.ts:9]:
```suggestion
```

File: cypress/e2e/grants_page_spec.cy.ts [hunk 6]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('Grants Page', function () {
-  testPageMetadata({ path: PATHS.GRANTS, useAbsoluteTitle: true})
+describe('Grants Page', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.GRANTS,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/grants_page_spec.cy.ts:9]:
```suggestion
```

File: cypress/e2e/home_page_spec.cy.ts [hunk 7]
```diff
@@ -1,6 +1,16 @@
...
 import { testPageMetadata } from '../support/test-utils'
+import { verifyLinks } from '../support/verifyLinksUtil'
 
-describe('Home Page', function () {
-  testPageMetadata({ path: PATHS.HOME, useAbsoluteTitle: true})
+describe('Homepage', () => {
+  it('should check metadata', () => {
+    testPageMetadata({
+      path: PATHS.HOME,
+      hasPageHeaderDescription: true,
```
Comment (mirhamasala) [inline cypress/e2e/home_page_spec.cy.ts:9]:
```suggestion
```

**Pull request merged**